packages/admin-panel/src/pages/ChangePasswordPage.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Tupaia MediTrak
```

Comment:
This file's not really part of this PR, but will hand wave it coz I think that's the next PR in the reviews channel